src/Treeview.ts

```diff
@@ -38,7 +38,13 @@ const resourceIcons: [string, string][] = [
     ["Microsoft.Compute/virtualMachines/extensions", "extensions.svg"],
     ["Microsoft.Network/networkSecurityGroups", "nsg.svg"],
     ["Microsoft.Network/networkInterfaces", "nic.svg"],
-    ["Microsoft.Network/publicIPAddresses", "publicip.svg"]
+    ["Microsoft.Network/publicIPAddresses", "publicip.svg"],
+    ["Microsoft.DocumentDB/databaseAccounts", "documentdb.svg"],
+    ["Microsoft.Web/serverfarms", "appserviceplan.svg"],
```

Comment:
appserviceplan also doesn't look great in dark theme. I don't have an svg readily available, but this is what it looks like for me in the portal:
![screen shot 2018-10-17 at 8 33 02 am](https://user-images.githubusercontent.com/11282622/47099339-f141b300-d1e9-11e8-964c-208b78681d48.png)